src/modules/sched-simple/sched.c

```diff
@@ -141,21 +141,25 @@ jobreq_create (const flux_msg_t *msg)
 
 static void simple_sched_destroy (flux_t *h, struct simple_sched *ss)
 {
```

Comment:
commit message typo: "retore"

Reply:
Thanks!  Fixed that and forced a push.  I'll set MWP.